components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -81,6 +81,18 @@ Refer to the following pages on how to create users and assign roles in ServiceN
 - [Creating Users in ServiceNow](https://docs.servicenow.com/csh?topicname=t_CreateAUser.html)
 - [Assigning Roles in ServiceNow](https://docs.servicenow.com/csh?topicname=t_AssignARoleToAUser.html)
 
+### Discovery Source
+
+The Chef Automate Integration App provides a unique ID as a discovery source that distinguishes between discovered Configuration Items (CIs) and imported CIs. The **Discovery Source** field is part of the `cmdb_servers` table, and it takes **ChefAutomate** value while importing CIs by default.
```

Comment:
Do you know the purpose of the `cmdb_servers` table? Is this a list of nodes/servers/anything that have been found by Discovery but haven't been added to the CMDB?

Reply:
Discovery source is a separate field, that identifies the sources. The CMDB table purpose we have given in the intro. This field is part of the CMDB table to give an identity whether the server is imported or belongs to discovered item.

Reply:
Ok, this is starting to make sense to me. I think this is correct.


```suggestion
The Chef Automate Integration App provides a unique ID that distinguishes between discovered Configuration Items (CIs) and imported CIs. Chef Infra Servers that are discovered by the Integration App are assigned a value of **ChefAutomate** in the Discovery Source field of the Chef Servers table.
```

Reply:
Am not sure whether its a Chef Infra server Ian? Shall I change them after confirming with the Eng. team?

Reply:
this is what eng. team says: instead of, Chef Infra Servers that are discovered by the Integration App; we can say,

Chef Infra Servers and other CIs that are discovered by the Integration App. Because apart from Chef Infra Server there are other CIs too like filesystem, network etc

Reply:
Ok, I updated my suggestion to just say "CIs that are discovered..." I think that's good.